src/rest/routes/enterprise/delete-ghe-server.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response } from "express";
+import { ParamsDictionary } from "express-serve-static-core";
+import { errorWrapper } from "../../helper";
+import { BaseLocals } from "..";
+import { GitHubServerApp } from "~/src/models/github-server-app";
+import { isConnected } from "~/src/util/is-connected";
+import { saveConfiguredAppProperties } from "~/src/util/app-properties-utils";
+import { InvalidArgumentError } from "~/src/config/errors";
+
+const deleteEnterpriseServer = async (
+	req: Request<ParamsDictionary, unknown>,
+	res: Response<string, BaseLocals>
+): Promise<void> => {
+	const { installation } = res.locals;
+
+	const cloudOrUUID = req.params.cloudOrUUID;
+	const gheUUID = cloudOrUUID === "cloud" ? undefined : cloudOrUUID; //TODO: validate the uuid regex
+
+	if (!gheUUID) {
+		throw new InvalidArgumentError(
+			"Invalid route, couldn't determine UUID of enterprise server!"
+		);
+	}
+
+	const gitHubApp = await GitHubServerApp.getForUuidAndInstallationId(
```

Comment:
Why do we need this `const gitHubApp = await GitHubServerApp.getForUuidAndInstallationId(` ?

Reply:
to fetch the GHE server url from UUID of rest api path
GHE server url is requied for 
GitHubServerApp.uninstallServer